apm-agent-plugins/apm-spring-webmvc-plugin/pom.xml

```diff
@@ -51,6 +51,12 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
What is this used for?

Reply:
without this I got exception:
```
Error:(34, 8) java: cannot access reactor.core.publisher.Mono
  class file for reactor.core.publisher.Mono not found
```

Reply:
![image](https://user-images.githubusercontent.com/48118512/62318512-d1038c00-b4bd-11e9-9e2f-fc5b05e3df57.png)